### ISSUE TITLE
NOTE about compiled code using std::cout

### ISSUE THREAD

**Issue by bbolker** [glmmTMB/glmmTMB#141]
We need to fix this if we're going to submit to CRAN.  Probably pretty easy to fix for somebody who knows what they're doing/where this might be called in the C++ code ...

```
* checking compiled code ... NOTE
File ‘glmmTMB/libs/glmmTMB.so’:
  Found ‘_ZSt4cout’, possibly from ‘std::cout’ (C++)
    Object: ‘glmmTMB.o’

Compiled code should not call entry points which might terminate R nor
write to stdout/stderr instead of to the console, nor the system RNG.
```

**Comment by kaskr** [comment]
I have it under control:
https://github.com/kaskr/adcomp/commits/cout
Will be fixed in the next CRAN submission of TMB.

**Comment by kaskr** [comment]
Solved in TMB version 1.6.5 (on CRAN).

**Comment by bbolker** [comment]
... and `glmmTMB` not passes R CMD check -- pretty exciting.
